src/XmlNotepad/FormMain.cs

```diff
@@ -276,6 +293,52 @@ public FormMain()
             this.Controls.SetChildIndex(this.resizer, 0);
             this.taskList.Site = this;
 
+            // populate default settings and provide type info.
```

Comment:
This used to all be in a separate function SetDefaultSettings(), what happened to that?

Reply:
It is the 2nd line in the FormMain constructor (line 244):
```C#
        public FormMain()
        {
            this.settings = new Settings();
            SetDefaultSettings();
            ...etc...
```
But the original version I forked in Feb already had it like that. I guess the change-author forgot to select the comment with the method call.. :)